build_tools/github/trigger_hosting.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+set -x
+
+GITHUB_RUN_URL=https://nightly.link/$GITHUB_REPOSITORY/actions/runs/$RUN_ID
+
+if [ "$EVENT" == pull_request ]
+then
+     BRANCH=pull/$PULL_REQUEST_NUMBER/head
+else
+     BRANCH=$HEAD_BRANCH
+fi
+
+curl --request POST \
+     --url https://circleci.com/api/v2/project/gh/$GITHUB_REPOSITORY/pipeline \
+     --header "Circle-Token: $CCI_TOKEN" \
+     --header "content-type: application/json" \
+     --header "x-attribution-actor-id: github_actions" \
+     --header "x-attribution-login: github_actions" \
+     --data \{\"branch\":\"$BRANCH\",\"parameters\":\{\"GITHUB_RUN_URL\":\"$GITHUB_RUN_URL\"\}\}
```

Comment:
That would probably require rewriting this `build_tools/github/trigger_hosting.sh` shell script into a Python script though.

Reply:
I think this can be a follow up PR. 

Looking at the [PR on the fork](https://github.com/alfaro96/scikit-learn/pull/1) the circleci link will display as one of the checks.